src/modules/Schema/containers/CreateSchema/useCreateSchema.tsx

```diff
@@ -1,26 +1,28 @@
 import { yupResolver } from '@hookform/resolvers/yup';
+import { t as translate } from 'i18next';
 import { useImperativeHandle, useState } from 'react';
 import { useForm, useFieldArray } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
 import { useNavigate } from 'react-router-dom';
+import { SCHEMA_ATTRIBUTES } from 'src/constants/SCHEMA';
 import { AttributeOption, createSchemaAdaptor, SchemaReq } from 'src/core/adaptors';
 import * as yup from 'yup';
 
 const schema = yup
   .object()
   .shape({
-    name: yup.string().required('This field is required'),
+    name: yup.string().required(translate('schema-required-error')),
```

Comment:
When you call translate out of the component, I've tested before it needs useEffect to recall it
because sometimes the error won't appear.
for more information, I will talk to you after the daily meeting on Monday,